server/src/controllers/post.controllers/update.post.by.id.ts

```diff
@@ -1,5 +1,17 @@
-import {NextFunction, Request, Response} from 'express';
+import { Request, Response, NextFunction } from 'express';
+import { updatePost } from '../../db.requests/updatePost.request';
+import { getTags } from '../../helpers/getTags.post';
 
-export const update = (req: Request, res: Response, next: NextFunction): void => {
-    // TODO it will be implemented later
+export const update = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
+    try {
+        const id = req.params.id;
+        const description = req.body.description;
+        const tags = getTags(description);
+
+        const updPost = await updatePost(id, description, tags, next);
```

Comment:
just on bd or img too?

Reply:
We decided, that user can only update a description of post, but not an image. The same functionality has Instagram. Or should we allow updating img too?

Reply:
Ok, that your decision😉